usaspending_api/disaster/v2/views/federal_account/federal_account_result.py

```diff
@@ -0,0 +1,68 @@
+from dataclasses import dataclass, field
+from dataclasses_json import dataclass_json
+from typing import Dict
+
+from usaspending_api.common.data_classes import Pagination
+from usaspending_api.disaster.v2.views.data_classes import Collation, Element
+
+
+class TAS(Element):
+    """Renaming the original generic object to clearly be TAS"""
+
+
+class FedAccount(Collation):
+    """Renaming the original generic object to clearly be Federal Account"""
+
+
+@dataclass_json
+@dataclass
+class FedAcctResults:
+    _federal_accounts: Dict[FedAccount, FedAccount] = field(default_factory=dict)
+
+    def __getitem__(self, key):
+        try:
+            return self._federal_accounts[key]
+        except KeyError:
+            # If the dictionary key doesn't exist, populate with the key like a DefaultDict
+            self._federal_accounts[key] = key
+            return self._federal_accounts[key]
```

Comment:
Can this entire try/except be replaced with `return self._federal_accounts.setdefault(key,  key)`?

Reply:
Yes